wof/wofpy_config.py

```diff
@@ -16,62 +16,82 @@
 
 Usage:
     wofpy_config INDIR
-    wofpy_config INDIR [--mode=<development,production>]
+    wofpy_config INDIR [--mode=<development,production>] [--overwrite=<soft,hard,none>]
 
-    wofpy_config (-h | --help | -v | --version)
+    wofpy_config (-h | --help | -v | --version | -m | --mode | -o | --overwrite)
 
 Examples:
     wofpy_config wofpyserver
+    wofpy_config wofpyserver --mode=production --overwrite=soft
 
 Arguments:
-  directory     Configuration directory.
+  directory               Configuration directory.
 
 Options:
-  -h --help     Show this screen.
-  -v --version  Show version.
-  --mode=development   deploy mode [default: development]
+  -h --help               Show this screen.
+  -v --version            Show version.
+  -m --mode=development   Deploympent mode [default: development].
+  -o --overwrite=none     Overwrite everything *hard*,
```

Comment:
`docopt` is quite powerful to create CLI. I prefer it to `click` or `fire` b/c it is only a POSIX parser on the `__doc__` text.